client/job.lua

```diff
@@ -114,7 +114,7 @@ end)
 ---Use first aid on nearest player to revive them.
 ---Intended to be invoked by client or server.
 RegisterNetEvent('hospital:client:RevivePlayer', function()
-    if not exports.qbx_core:HasItem('firstaid') then
+    if not HasItem('firstaid') then
```

Comment:
This should be using the ox_inventory Search. While this functionally works this is a deprecated function that may be removed in the future.

https://overextended.dev/ox_inventory/Functions/Client#search